### ISSUE TITLE
Included schema does not work

### ISSUE THREAD

**Issue by timur-han** [mulesoft/osprey#151]
Hello,

I include a schema on my path using the following directive: 
```
UserMessage:
      type: !include schemas/user-management.json
```
However it is somehow not used during the validation. I receive the following error:

"errors":[{"type":"json","keyword":"type","schema":"unknown","data":{"userManagementMessage":{"messageType":"a"}},"message":"Invalid type expected unknown"}],"stack":"BadRequestError: Request failed to validate against RAML definition\n    at createValidationError

When I replace the type with a RAML type it works smoothly. What can be the problem?

**Comment by jstoiko** [comment]
This seems to be an issue with `osprey-method-handler` which makes some gymnastic with `type` and `schema` properties.

A workaround would be to either use a RAML type like you mentioned or use the `schema` property. 

E.g.
```
#%RAML 1.0
title: My api
baseUri: http://localhost:3000

/foo:
  post:
    body:
      application/json:
        schema: !include schemas/foo.json

```

**Comment by jstoiko** [comment]
This is fixed in `v0.5.0`
